explorer/nodewatch/nodewatch/NetworkConnector.py

```diff
@@ -63,9 +65,19 @@ async def _update_height(self, session, descriptor):
 				if 'height' in response_json:
 					descriptor.height = int(response_json['height'])
 
-				if 'finalized_height' in response_json:
-					descriptor.finalized_height = int(response_json['finalized_height'])
-		except (aiohttp.client_exceptions.ClientConnectorError, aiohttp.client_exceptions.ContentTypeError, asyncio.TimeoutError) as ex:
+				if 'latestFinalizedBlock' in response_json:
+					descriptor.finalized_info = FinalizedInfo(
+						int(response_json['latestFinalizedBlock'].get('height', 0)),
```

Comment:
1. would put `response_json['latestFinalizedBlock']` in local variable
2. are there cases where subfield is not available (i.e. why using get with default parameter)?